src/controllers/AuthController.ts

```diff
@@ -0,0 +1,26 @@
+import express from "express";
+import { getToken } from "../services/AuthService";
+
+export const getLoginForm = async (req: express.Request, res: express.Response): Promise<void> => {
+    res.render('loginForm.html');
+}
+
+export const postLoginForm = async (req: express.Request, res: express.Response): Promise<void> => {
+    try {
+        req.session.token = await getToken(req.body);
+        res.redirect('/homepage');
+    } catch (e) {
+        res.locals.errormessage = e.message;
+        res.render('loginForm.html', { ...req.body, errormessage: res.locals.errormessage });
+    }
+}
+
+export const logout = async (req: express.Request, res: express.Response): Promise<void> => {
+    req.session.destroy((err) => {
+        if (err) {
+            res.status(500).send('Failed to log out');
+        } else {
+            res.status(200).send('Logged out successfully');
```

Comment:
redirect to login

Reply:
<img width="691" alt="Screenshot 2024-07-31 at 13 57 42" src="https://github.com/user-attachments/assets/de0e62b5-6207-4c45-80b8-acb145d7f968">

We are currently using the html to redirect to the login page, do you still want this done in the controller?